src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -50,7 +51,8 @@ class AmendmentTableImpl : public AmendmentTable
     core::Clock::time_point m_lastReport;  // close time of most recent majority report
     beast::Journal m_journal;
 
-    AmendmentState* getCreate (uint256 const& amendment, bool create);
+    AmendmentState& getCreate (uint256 const& amendment);
+    AmendmentState* getExisting (uint256 const& amendment);
```

Comment:
Its good as written. The different return types are no big deal. The alternative is to return an iterator and provide begin/end functions. Which we should consider if significant new code calls the public interfaces of this class.